src/nvim/screen.c

```diff
@@ -4512,23 +4512,23 @@ void screen_adjust_grid(ScreenGrid **grid, int *row_off, int *col_off)
 // @param[out] n_extrap number of characters from pp_extra to display
 // @param[in, out] sign_idxp Index of the displayed sign
 static void get_sign_display_info(
-    bool nrcol,
-    win_T *wp,
-    sign_attrs_T sattrs[],
-    int row,
-    int startrow,
-    int filler_lines,
-    int filler_todo,
-    int count,
-    int *c_extrap,
-    int *c_finalp,
-    char_u *extra,
-    size_t extra_size,
-    char_u **pp_extra,
-    int *n_extrap,
-    int *char_attrp,
-    int *draw_statep,
-    int *sign_idxp
+                                  bool nrcol,
+                                  win_T *wp,
+                                  sign_attrs_T sattrs[],
+                                  int row,
+                                  int startrow,
+                                  int filler_lines,
+                                  int filler_todo,
+                                  int count,
+                                  int *c_extrap,
+                                  int *c_finalp,
+                                  char_u *extra,
+                                  size_t extra_size,
+                                  char_u **pp_extra,
+                                  int *n_extrap,
+                                  int *char_attrp,
+                                  int *draw_statep,
+                                  int *sign_idxp
```

Comment:
An example of what I mentioned [here](https://github.com/neovim/neovim/pull/15465/files#r693993300).

Reply:
Yep. Not really a fan myself if I'm being honest.

Reply:
Ok it looks a little less dumb when using the "real" neovim style. I'll show in a sec.